package.json

```diff
@@ -4,39 +4,56 @@
   "description": "Simple application development in Rails",
   "main": "dist/index.cjs.js",
   "module": "dist/index.esm.js",
-  "types": "dist/index",
+  "types": "dist/index.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "build": "yarn build:js && yarn build:types",
+    "build": "yarn build:js && yarn build:components && yarn build:types",
     "build:js": "NODE_ENV=production rollup -c",
+    "build:components": "NODE_ENV=production rollup --config rollup.component.config.js && echo 'import \"./stack.css\";' >> dist/components/stack.js",
     "build:types": "tsc --emitDeclarationOnly",
     "format": "prettier --write \"src/**/*.{ts,css,scss,json,yml}\"",
     "lint": "yarn lint:types && yarn lint:style",
     "lint:types": "tsc --noEmit",
     "lint:style": "eslint src/**/*.ts --max-warnings 0"
   },
   "contributors": [
-    "Jens Ravens"
+    "Jens Ravens",
+    "Geraldine Atayan"
   ],
   "license": "MIT",
   "dependencies": {
     "@hotwired/stimulus": "^3.0.1",
     "@popperjs/core": "^2.11.0",
-    "@rails/request.js": "^0.0.6"
+    "@rails/request.js": "^0.0.6",
+    "classnames": "^2.3.2",
+    "react": "^17.0.1",
```

Comment:
those should be peer dependencies. I don't want to include react in my rails apps.

Reply:
moved react and react-dom to peer dependencies